shared/p2p/options.go

```diff
@@ -2,35 +2,82 @@ package p2p
 
 import (
 	"fmt"
+	"io/ioutil"
 	"net"
+	"os"
+
+	peer "github.com/libp2p/go-libp2p-peer"
 
 	"github.com/libp2p/go-libp2p"
+	crypto "github.com/libp2p/go-libp2p-crypto"
+	"github.com/prysmaticlabs/prysm/shared/iputils"
+
```

Comment:
Please fix this import grouping. It shuold be two groups: std libraries and everything else.